### ISSUE TITLE
Write Server Setup Documentation

### ISSUE THREAD

**Issue by dwgibbons** [col-bsa/Unami-Lodge-Documentation#6]
**Owner / Committee:**
Communications

**Due Date:**
1/16/18

**Description:**
Write documentation on server setup and configuration

**Tasks:** 
- [x] Write initial server setup instructions
- [ ] List up-to-date divergences from the documentation

**Comment by dwgibbons** [comment]
I guess now this needs to be updated since we have a new build.

**Comment by crossfireprod** [comment]
Yeah, and the server setup scripts really need to be able to ingest *all* of the config.  (ex. Repo names and keys.)
